data/data-pipeline/data_pipeline/etl/sources/census_acs/etl.py

```diff
@@ -77,7 +77,7 @@ def extract(self) -> None:
                     src="acs5",
                     year=self.ACS_YEAR,
                     geo=censusdata.censusgeo(
-                        [("state", fips), ("county", "*"), ("block group", "*")]
+                        [("state", fips), ("county", "*"), ("tract", "*")]
```

Comment:
@switzersc-usds -- do we also need to change this line from `etl_score.py`: 

Move `self.census_df` from 

```
census_block_group_dfs = [
            self.ejscreen_df,
            self.census_df,
            self.housing_and_transportation_df,
            self.census_acs_median_incomes_df,
            self.national_risk_index_df,
        ]
```

to move `self.census_df` into `census_tract_dfs`?

Reply:
I was just looking at that. I could do that all at once, or move each df over to the tracts df list as I go. Probably moving them over as I go makes sense, so we're keeping track of them.

Reply:
Right - and moving it as we go means that the score should (🤞 ) still work even as we're moving1

Reply:
Updated